--- a/SConstruct	2014-11-01 21:36:53.988229447 +0100
+++ b/SConstruct	2014-11-01 21:38:58.284233425 +0100
@@ -283,8 +283,12 @@
 if env['TARG'] != 'stm32':
     if env['PLATFORM'] == 'win32':
         env.Append(LIBS = ['ajtcl_st'])
-    elif env['PLATFORM'] == 'posix' or env['PLATFORM'] == 'darwin':
-        env.Prepend(LIBS = ['libajtcl_st'])
+
+    if env['PLATFORM'] == 'posix':
+        env.Append(LIBS = ['libajtcl'])
+		
+    if env['PLATFORM'] == 'darwin':
+        env.Append(LIBS = ['libajtcl'])
 
     progs = env.SConscript('SConscript', 'env', variant_dir='build/$VARIANT', duplicate=0)
 else:
